packages/react-lowcode/src/codegen/ast/jsxElements.ts

```diff
@@ -0,0 +1,9 @@
+import ts, { factory } from "typescript";
+
+export function createJsxSelfclosingElement(tagName: string): ts.JsxSelfClosingElement{
+    return factory.createJsxSelfClosingElement(
```

Comment:
we allready have this wrapper https://github.com/iteria-app/lowcode/blob/4c96212aa6fbf5b430edbd34c4fd8242aa762eb1/packages/react-lowcode/src/ast/tags.ts#L14